docs/pages/installation.md

```diff
@@ -158,6 +250,21 @@ Here's what comes in the package.
 
```

Comment:
I think this could use some additional work... if someone is brand new, we should clarify to them when they'd use SASS, when they'd use CSS, and when they'd use dist vs the other files, and maybe link them to resources on those. Right now if I didn't have prior knowledge I'd be totally confused about why there's both `js` and `dist/js`, as well as whether I was supposed to use `scss` or `dist/css` or both